src/SFA.DAS.Support.Indexer.ApplicationServices.UnitTests/app.config

```diff
@@ -7,6 +7,14 @@
         <bindingRedirect oldVersion="0.0.0.0-10.0.0.0" newVersion="10.0.0.0" />
       </dependentAssembly>
       
+      <dependentAssembly>
+        <assemblyIdentity name="System.Diagnostics.DiagnosticSource" publicKeyToken="cc7b13ffcd2ddd51" culture="neutral" />
+        <bindingRedirect oldVersion="0.0.0.0-4.0.3.1" newVersion="4.0.3.1" />
```

Comment:
should the new version be 5.0.0, as it in SFA.DAS.Support.Portal.UnitTests/app.config